### ISSUE TITLE
Unable to find Flash plugin in Ubuntu 16.04

### ISSUE THREAD

**Issue by lbssousa** [tiliado/nuvolaplayer#236]
I've installed Nuvola Player 3.0 in my Ubuntu MATE 16.04 64-bit box, and I'm trying to use Deezer on it. However, it gives me a warning that it's unable to find my Flash plugin. I've tried both Ubuntu packages `flashplugin-installer` and `adobe-flashplugin`, without success.

I'm not sure if it's a problem with Nuvola Player itself or with WebKitGTK+.

Here is my `nuvolaplayer3 -v` output:

```
Master: [WARNING    Gtk] Theme parsing error: gtk-main.css:69:33: Failed to import: Erro ao abrir arquivo: No such file or directory
Master: [INFO     Nuvola] WebAppRegistry.vala:89: Using web app Deezer, version 2.4, data dir /usr/share/nuvolaplayer3/web_apps/deezer
Runner: [WARNING    Gtk] Theme parsing error: gtk-main.css:69:33: Failed to import: Erro ao abrir arquivo: Arquivo ou diretório não encontrado
Runner: [WARNING    Gtk] Theme parsing error: gtk-main.css:69:33: Failed to import: Erro ao abrir arquivo: Arquivo ou diretório não encontrado
Runner: [WARNING  GLib-GObject] g_object_set_property: construct property "local-storage-directory" for object 'WebKitWebContext' can't be set after construction
Runner: 
(WebKitWebProcess:24591): Gtk-WARNING **: Theme parsing error: gtk-main.css:69:33: Failed to import: Erro ao abrir arquivo: Arquivo ou diretório não encontrado
Worker: [DEBUG    Nuvola] WebExtension.vala:242: Page 1 created for (null)
Worker: [DEBUG    Nuvola] WebExtension.vala:192: Waiting until initialized
Runner: [WARNING  Nuvola] FormatSupportCheck.vala:152: No Flash plugin has been found.
Worker: [DEBUG    Nuvola] WebExtension.vala:136: Window object cleared for 'http://www.deezer.com/profile/243598713'
Worker: [DEBUG    Nuvola] WebExtension.vala:144: Init frame: http://www.deezer.com/profile/243598713, 0x55c5e2ff4220, 0x55c5e2e4ca20, 0x7f9bec28f740
Runner: [CRITICAL  Json] json_node_get_parent: assertion 'JSON_NODE_IS_VALID (node)' failed
Runner: [CRITICAL  Json] Application will not function properly.
Runner: [CRITICAL  GLib] g_ptr_array_remove_index: assertion 'index_ < rarray->len' failed
Runner: [CRITICAL  GLib] Application will not function properly.
Runner: [CRITICAL  Json] json_node_get_parent: assertion 'JSON_NODE_IS_VALID (node)' failed
Runner: [CRITICAL  Json] Application will not function properly.
Runner: [CRITICAL  GLib] g_ptr_array_remove_index: assertion 'index_ < rarray->len' failed
Runner: [CRITICAL  GLib] Application will not function properly.
Runner: [INFO     Nuvola] AppRunnerController.vala:236: Logged in as  ()
Runner: http://connect.facebook.net/pt_BR/sdk/vb.js:39:260: CONSOLE ERROR Flash is not installed or is too old
```

**Comment by fenryxo** [comment]
Hello @lbssousa. It might be same issue as in #231. Could you install the `libwebkit2gtk-4.0-37-gtk2` package and try again? Ubuntu Mate doesn't install this package by default unlike Ubuntu with Unity. Packaging will be fixed in the next release #232.

**Comment by lbssousa** [comment]
It did the trick! Thank you very much!

BTW you can mark this issue as a duplicate of #231. Strangely I was unable to find it by searching at Github.

**Comment by fenryxo** [comment]
The fix has been released in Nuvola Player 3.0.3. Thanks for your bug report.
